File: placementprepration/linkedlidt.java
public class linkedlidt{
    private Node head;
    private Node tail;
    public linkedlidt(){
        this.size=0;
    }

    private class Node{
        private int value;
        private Node next;
        public Node(int value){
            this.value=value;
        }
        public Node(int value,Node next){
            this.value=value;
            this.next=next;
        }
    }
    public static void main(String[] args) {
        
    }
}

File: placementprepration/linkedlidt.py
class Node:
    def __init__(self, value, nxt=None):
        self.value = value
        self.next = nxt


class LinkedList:
    def __init__(self):
        self.head = None
        self.tail = None
        self.size = 0

    def append(self, value):
        node = Node(value)
        if not self.head:
            self.head = self.tail = node
        else:
            self.tail.next = node
            self.tail = node
        self.size += 1


if __name__ == "__main__":
    ll = LinkedList()
    # Add interactive methods here if needed
